### PR TITLE
Some random cleanups (and probably a bug-fix)

### PR DESCRIPTION
**Pull request by oscargus**
* Removed unused imports
* Some code style fixes
* Most likely fixed a potential NPE in SaveDataBaseAction (chosenFile will never be null here, but f might be and is later dereferenced)

### CODE REVIEW

**Comment by simonharrer** [comment]
:+1:

File: src/main/java/net/sf/jabref/gui/dbproperties/FieldFormatterCleanupsPanel.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
     private JButton resetButton;
 
-    private FieldFormatterCleanups defaultFormatters;
+    private final FieldFormatterCleanups defaultFormatters;
```
Comment (Siedlerchr) [inline src/main/java/net/sf/jabref/gui/dbproperties/FieldFormatterCleanupsPanel.java]: Could you please exclude this file from your commit as I am currently working with this file for #957  and #958?
I included your changes already.

Comment (oscargus) [inline src/main/java/net/sf/jabref/gui/dbproperties/FieldFormatterCleanupsPanel.java]: Sure, it was only line 233 which I personally changed, Eclipse took care of the rest.

**Comment by oscargus** [comment]
Reverted the change the @Siedlerchr asked for, but did a force push so that it wasn't changed back and forth...

File: src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java [hunk 2]
```diff
@@ -68,12 +67,12 @@ public boolean accept(File f) {
...
-        addEntyDataFromXMP(pdfFile, entry);
-
+        addEntryDataFromXMP(pdfFile, entry);
+        
         if (entry.getField("title") == null) {
         	entry.setField("title", pdfFile.getName());
         }
-
+        
         return entry;*/
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java]: I would suggest just delete this commented out code.

Comment (oscargus) [inline src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java]: Maybe. However, in that case we can also remove addEntryDataFromPDDocumentInformation and  addEntryDataFromXMP (if I recall correctly) and it seems like those could be useful if they are used. Of course we can recreate those if/when needed.

Comment (simonharrer) [inline src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java]: Ok, then we leave it in for now.

**Pull request merged**